src/components/onboardingGuide/onboardingGuide.scss

```diff
@@ -16,13 +16,19 @@
 
     .onboarding__bottom {
         height: calc(100vh - 300px);
-        .onboarding-card a:hover {
-            color: var(--N900);
-            text-decoration: none;
+
+        .onboarding-card a {
```

Comment:
this and next class can be merged together

Reply:
done